data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -50,9 +50,8 @@ from dimcand
         select distinct on (cand_sk) * from dimcandproperties
             order by cand_sk, candproperties_sk desc
     ) cand_p_most_recent using(cand_sk, cand_id)
-    left join dimcandstatusici csi_all using (cand_sk)
-    left join dimcandoffice co on co.cand_sk = dimcand.cand_sk and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)  -- only joined to get to dimoffice
-    inner join dimoffice using (office_sk)
+    left join dimcandoffice co on co.cand_sk = dimcand.cand_sk and (csi_recent.election_yr is null or co.cand_election_yr = csi_recent.election_yr)
```

Comment:
Could we drop the `and...` here?
`left join dimcandoffice co on co.cand_sk = dimcand.cand_sk`